subsys/bluetooth/controller/ll_sw/lll_chan.c

```diff
@@ -147,15 +147,9 @@ uint8_t lll_chan_iso_subevent(uint16_t chan_id, uint8_t *chan_map,
 		x = 0;
 	}
 
-	chan_idx = ((((uint32_t)prn_subevent_se * x) >> 16) +
-		    d + *remap_idx) % chan_count;
-
-	if ((chan_map[chan_idx >> 3] & (1 << (chan_idx % 8))) == 0U) {
-		*remap_idx = chan_idx;
-		chan_idx = chan_sel_remap(chan_map, *remap_idx);
-	} else {
-		*remap_idx = chan_idx;
-	}
```

Comment:
This has been tested by `lll_chan_sel_2_ut()` with the sample data in the specification. Please add a testcase that fails with the implementation.

Reply:
Yes, I know it’s tested against reference data from spec. Unfortunately, the sample data in spec. is poorly chosen. For a mapped ch; as long as the ch_idx and the remap_idx are identical the issue will not reveal itself. We need a different ch map to trigger the issue.
Point taken; I’ll provide a test catching the issue.

Reply:
I have added 2 used channels channel map to cover the change.